### PR TITLE
Feat zawolanski

### PR DESCRIPTION
**Pull request by zawolanski**

### CODE REVIEW

File: src/theme/GlobalStyles.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalStyles = createGlobalStyle`
+    body {
+        margin: 0;
```
Comment (pawelsobkowski) [inline src/theme/GlobalStyles.js:5]: Also, I would add `padding: 0` and `box-sizing: border-box`

```suggestion
        margin: 0;
        padding: 0;
        box-sizing: border-box;
```

Comment (zawolanski) [reply src/theme/GlobalStyles.js:5]: I thought about add something like this [styled-normalize](https://github.com/sergeysova/styled-normalize). What do you think?

Comment (pawelsobkowski) [reply src/theme/GlobalStyles.js:5]: Why do you want it? I think we can achieve the same result without it and additional tags in code like this:
```
<React.Fragment>
    <Normalize />
    <App />
</React.Fragment>
```

Comment (zawolanski) [reply src/theme/GlobalStyles.js:5]: Ok, we can do it ourselves

Comment (pawelsobkowski) [reply src/theme/GlobalStyles.js:5]: I looked at that github but i didn't see any pros of that solution

**Review by pawelsobkowski — APPROVED** [review]

**Pull request merged**